### DISCUSSION TITLE
MBEDTLS_CIPHER_MODE_CFB required

### DISCUSSION THREAD

**Discussion by yyshell** [immortalwrt/immortalwrt#1288] [General]
rockchip r4s build failed, messages:
checking whether mbedtls supports Cipher Feedback mode or not... configure: error: MBEDTLS_CIPHER_MODE_CFB required
make[3]: *** [Makefile:130: /home/adam/immortalwrt/build_dir/target-aarch64_generic_musl/shadowsocks-libev-3.3.5/.configured_68b329da9893e34099c7d8ad5cb9c940] Error 1
make[3]: Leaving directory '/home/adam/immortalwrt/feeds/packages/net/shadowsocks-libev'
time: package/feeds/packages/shadowsocks-libev/compile#23.94#1.51#28.29
    ERROR: package/feeds/packages/shadowsocks-libev failed to build.
make[2]: *** [package/Makefile:129: package/feeds/packages/shadowsocks-libev/compile] Error 1
make[2]: Leaving directory '/home/adam/immortalwrt'
make[1]: *** [package/Makefile:123: /home/adam/immortalwrt/staging_dir/target-aarch64_generic_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/adam/immortalwrt'
make: *** [/home/adam/immortalwrt/include/toplevel.mk:233: world] Error 2

**Comment by 1715173329** [comment]
Use ss-rust instrad.

**Reply by yyshell** [reply]
I build the image with passwall ,passwall2 and ssrP. the errors still occurs while I uncheck the shadowsocks-libev options.

**Comment by yyshell** [comment]
I build the r4s image successfully by delete all the shadows-libev in the .config file. Since shadowsocks-libev isn't used by many people anymore, shoud we  disable shadows-libev in the default configuration?
